golang.md

```diff
@@ -0,0 +1,119 @@
+# Golang at Clever
+
+## Golang versions
+
+All golang services (with some exceptions) at Clever should be on go-1.5.
+
+
+## Docker Images
+
+#### drone
+
+`clever/drone-go:1.5`
+
+#### runtime
+
+  - `gliderlabs/alpine:3.2` - see [catapult/Makefile](https://github.com/Clever/catapult/blob/master/Makefile) as an example. (extra build configuration needed)
+  - `google/debian:wheezy`
+
+
+## Dependencies
```

Comment:
It could be easy enough to link to [this Google Doc on Godeps](https://docs.google.com/document/d/1YZg2S7v1bir3MG1YvswAD2Y0KrsuDr-NCr6pPG2ycEM/edit).